src/main/java/com/arpnetworking/metrics/incubator/impl/TsdPeriodicMetricsFactory.java

```diff
@@ -0,0 +1,251 @@
+        }
+
+        @Override
+        public void incrementCounter(@Nonnull final String name, final long value) {
+            _wrapped.incrementCounter(name, value);
+        }
+
+        @Override
+        public void decrementCounter(@Nonnull final String name) {
+            _wrapped.decrementCounter(name);
+        }
+
+        @Override
+        public void decrementCounter(@Nonnull final String name, final long value) {
+            _wrapped.decrementCounter(name, value);
+        }
+
+        @Override
+        public void resetCounter(@Nonnull final String name) {
+            _wrapped.resetCounter(name);
+        }
+
+        @Override
+        public Timer createTimer(@Nonnull final String name) {
+            return _wrapped.createTimer(name);
+        }
+
+        @Override
+        public void startTimer(@Nonnull final String name) {
+            _wrapped.startTimer(name);
+        }
+
+        @Override
+        public void stopTimer(@Nonnull final String name) {
+            _wrapped.stopTimer(name);
+        }
+
+        @Override
+        public void setTimer(@Nonnull final String name, final long duration, @Nullable final TimeUnit unit) {
+            _wrapped.setTimer(name, duration, unit);
+        }
+
+        @Override
+        public void setTimer(@Nonnull final String name, final long duration, @Nullable final Unit unit) {
+            _wrapped.setTimer(name, duration, unit);
+        }
+
+        @Override
+        public void setGauge(@Nonnull final String name, final double value) {
+            _wrapped.setGauge(name, value);
+        }
+
+        @Override
+        public void setGauge(@Nonnull final String name, final double value, @Nullable final Unit unit) {
+            _wrapped.setGauge(name, value, unit);
+        }
+
+        @Override
+        public void setGauge(@Nonnull final String name, final long value) {
+            _wrapped.setGauge(name, value);
+        }
+
+        @Override
+        public void setGauge(@Nonnull final String name, final long value, @Nullable final Unit unit) {
+            _wrapped.setGauge(name, value, unit);
+        }
+
+        @Override
+        public void addAnnotation(@Nonnull final String key, @Nonnull final String value) {
+            _wrapped.addAnnotation(key, value);
+        }
+
+        @Override
+        public void addAnnotations(@Nonnull final Map<String, String> map) {
+            _wrapped.addAnnotations(map);
+        }
+
+        @Override
+        public boolean isOpen() {
+            return _wrapped.isOpen();
+        }
+
+        @Override
+        public void close() {
+            // Don't do anything. We don't want this instance to be closeable.
+        }
+
+        @Override
+        @Nullable
+        public Instant getOpenTime() {
+            return _wrapped.getOpenTime();
+        }
+
+        @Override
+        @Nullable
+        public Instant getCloseTime() {
+            return _wrapped.getCloseTime();
```

Comment:
This probably should not apply either. I understand why you have it in the incubator. But we should include additional inline documentation (for ourselves and users) explaining how we expect this to evolve.